doc/sphinx/Clusters_from_Scratch/active-passive.rst

```diff
@@ -241,6 +244,9 @@ gets going on the node, but it eventually will look like the below.)
       pacemaker: active/disabled
       pcsd: active/enabled
 
+
+.. index:: stickiness
```

Comment:
Should this be
```
.. index::
   single: stickiness
```

There are a couple other places where this could be changed, if so.

Reply:
It's an equivalent short-hand syntax when there are only single words to index (though comma-separated if more than one, which is confusing compared to "single:" with subtopics or "pair:")